crates/rust-analyzer/src/reload.rs

```diff
@@ -174,7 +174,11 @@ impl GlobalState {
 
         if let Err(_) = self.fetch_workspace_error() {
             status.health = lsp_ext::Health::Error;
-            message.push_str("Failed to load workspaces.\n\n");
+            message.push_str("Failed to load workspaces.");
+            if self.config.has_linked_projects() {
+                message.push_str(" Note that linked projects have been specified, it looks like this setting has been misconfigured.");
```

Comment:
if it's alright with you, I've hacked together a change to this that includes the paths of _which_ linked projects have been specified. Would you want to me push to a separate PR with that change?

Reply:
Feel free to PR that instead. I didn't bother putting much effort into this (just noticed I still had myself assigned to the issue :)